README.md

```diff
@@ -610,14 +611,14 @@ A startup goes through [several stages of growth](#stages-of-a-startup) as it ra
 based on the hope and expectation that the company will grow and make more money in the
 future.
 
-- 🄳 Companies add (or “issue”) shares during fundraising, which can then be awarded to
+- 🄳 Companies add (or issue) shares during fundraising, which can then be awarded to
```

Comment:
Precision nit: We're trying to give an informal synonym, but it now sounds like issuing might be a different thing from adding that companies also might do. Can be handled with quotes or italics, or making it formal by saying "This is called **issuing** shares."

Reply:
Right...  Fixed.

Reply:
I'm reverting this to quotes for now, b/c it'll be confusing to define two different terms in this section.